src/ILVerify/src/ILImporter.Verify.cs

```diff
@@ -334,6 +334,39 @@ void CheckIsAssignable(StackValue src, StackValue dst)
                 VerificationError(VerifierError.StackUnexpected, src, dst);
         }
 
+        private TypeDesc CheckIsAddressCompatibleWithType(StackValue address, TypeDesc type)
+        {
+            CheckIsByRef(address);
+
+            if (type != null)
+            {
+                CheckIsAssignablePointer(address.Type, type);
+            }
+            else
+            {
+                type = address.Type;
+                CheckIsObjRef(type);
+            }
+
+#if false
+            if (ptr.IsByRef())
```

Comment:
Could you please delete this `#if false` block while you are on it? It is left-over copy&paste from the JIT or PEVerify. It does not need to be here anymore because of it was implemented above already.